github/reactions.go

```diff
@@ -44,10 +44,21 @@ func (r Reaction) String() string {
 	return Stringify(r)
 }
 
+// ListCommentReactionOptions specifies the optional parameters to the
+// ReactionsService.ListCommentReactions method.
+type ListCommentReactionOptions struct {
+	// Content allows to return a single reaction type.
+	Content string `url:"content,omitempty"`
+
+	// ListOptions.Page has no effect.
```

Comment:
These two comment lines don't apply to this endpoint, so let's please remove them and leave `ListOptions` by itself on line 53 (and still retain the blank line on 52).  You will see examples of this use many times throughout this repo.